keras_nlp/tokenizers/word_piece_tokenizer_trainer.py

```diff
@@ -97,18 +100,21 @@ def compute_word_piece_vocabulary(
     >>> inputs.map(tokenizer.tokenize)
 
     Custom Split Usage (from filenames).
-    >>> def normalize_and_split(x):
-    ...     "Strip punctuation and split on whitespace."
-    ...     x = tf.strings.regex_replace(x, r"\p{P}", "")
-    ...     return tf.strings.split(x)
-    >>> with open("test.txt", "w+") as f: f.write("bat sat: pat mat rat.\n")
-    >>> inputs = tf.data.TextLineDataset(["test.txt"])
-    >>> split_inputs = inputs.map(normalize_and_split)
-    >>> vocab = compute_word_piece_vocabulary(split_inputs, 13, split=False)
-    >>> vocab
+    ```python
+    def normalize_and_split(x):
+        "Strip punctuation and split on whitespace."
+        x = tf.strings.regex_replace(x, r"\p{P}", "")
+        return tf.strings.split(x)
+    with open("test.txt", "w+") as f:
+        f.write("bat sat: pat mat rat.\n")
+    inputs = tf.data.TextLineDataset(["test.txt"])
+    split_inputs = inputs.map(normalize_and_split)
+    vocab = compute_word_piece_vocabulary(split_inputs, 13, split=False)
+    vocab
```

Comment:
remove this and the next line, since we don't actually have the print statement for the block examples

Reply:
Done!

Reply:
I think you remove the wrong two lines. Please remove

vocab = compute_word_piece_vocabulary(split_inputs, 13, split=False)
vocab

Reply:
Oh right sorry about that 😅😅